samples/README_Windows.md

```diff
@@ -0,0 +1,98 @@
+# Building Open Enclave SDK Samples on Windows
+
+All the samples that come with the Open Enclave SDK installation share similar directory structure and build instructions. The section contains general information on how to setup/build/sign/run all samples. It's important that you read information on this page before jumping into any individual sample.
+
+## Common Sample information
+
+### How Sample source code directories were structured
+
+Open Enclave SDK helps developers build enclave applications. An enclave application is partitioned into an untrusted component (called a host) and a trusted component (called an enclave). An enclave is a secure container whose memory (text and data) is protected from access by outside entities, including the host, privileged users, and even the hardware. All functionality that needs to be run in a Trusted Execution Environment (TEE) should be compiled into the enclave binary. The enclave may run in an untrusted environment with the expectation that secrets will not be compromised. A host is a normal user mode application that loads an enclave into its address space before starting interacting with an enclave. 
```

Comment:
nit: Please split source lines to have a reasonable length, to make reviewing/diffs easier going forward.